Dockerfile

```diff
@@ -17,15 +17,18 @@ RUN set -x \
  && curl -fsSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash -s -- --version "v${VERSION_HELM}" \
  && helm plugin install https://github.com/databus23/helm-diff \
  && python3 -m pip install --disable-pip-version-check --no-cache-dir \
-        ansible==8.4.0 \
-        ansible-core==2.15.4 \
-        Jinja2==3.1.3 \
-        netaddr==1.1.0 \
+        ansible-core==2.18.7 \
+        ansible==11.7.0 \
+        # unfortunately we cannot go to the latest bcrypt because passlib uses it wrong and it throws some confusing output, see: https://github.com/pyca/bcrypt/issues/684
+        bcrypt==4.0.1 \
         humanfriendly==10.0 \
+        Jinja2==3.1.3 \
         jmespath==1.0.1 \
-        kubernetes==25.3.0 \
-        pyjwt==2.8.0 \
+        kubernetes==33.1.0 \
+        netaddr==1.1.0 \
+        opencontainers==0.0.14 \
```

Comment:
Why do we need opencontainers?

Reply:
This can be used for downloading our release vectors and ansible-roles through signed OCI artifacts. Alse see: https://github.com/metal-stack/ansible-common/issues/33.